_pages/packaging-resources.md

```diff
@@ -0,0 +1,112 @@
+excerpt: "We make the scientific Python package ecosystem easier to navigate through peer review and packaging resources."
+header:
+  overlay_image: images/header.jpg
+  overlay_filter: 0.8
+intro:
+  - excerpt: "There are many tools and approaches that can be used to create a Python package. We build resources that help you both understand the tool ecosystem and also learn how to create a Python package using modern best practices."
+peer-review:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people from different backgrounds sitting around it, working on laptops and also writing together."
+    excerpt: "We run an open peer review process for scientific Python software. Peer review helps maintainers improve the quality, usability and long-term maintainability of scientific software."
+    title: Raising the bar for scientific Python software"
+    url: https://www.pyopensci.org/about-peer-review/index.html
+    btn_label: Learn more and get involved with peer review today.
+    btn_class: btn--primary
+packaging-easier:
+   - image_path: images/pyopensci-working-together.png
+     alt: "A black and grey sketch of a group of people sitting at a desk in front of a monitor smiling."
+     title: "Get Involved: Help us improve our scientific Python packaging resources"
+     excerpt: "
+     We need your help! Our packaging content is community-created and reviewed through an open review process on GitHub. The more feedback that we get, the more useful our resources are to the community. Get involved by:
+
+     * Opening an issue about problems that you find in our guidebook
+
+     * Submitting a pull request that fixes a typo or mistake in the guide.
+
+     * Get credit for your contribution
+
+
+     All contributions are recognized both on our website and in the guidebook's citation.
+     "
+     url: https://github.com/pyOpenSci/python-package-guide/pulls
+     btn_label: See what we're working on now
+     btn_class: btn--primary
+community-created:
+   - image_path: images/pyopensci-people-working-together.png
+     alt: "A group of people working on a tetris like set of building blocks, trying to stack the blocks together. "
+     title: "Community created Python Packaging Guide"
+     excerpt: "
+     Our packaging guide is a living document that will help you navigate the Python packaging ecosystem and also learn about modern Python packaging best practices.
+
+    * Created & curated by the community
+
+    * Reviewed by beginner to expert level Pythonistas
+
+    * Accurate, modern & beginner-friendly
+
+
+     All contributions are recognized both on our website and in the guidebook's citation.
+     "
+toc: false
+---
+
+{% include feature_row id="intro" type="center" %}
+
+<div class="pyos-section purple">
+<div class="content" markdown="1">
+
+{% include feature_row id="community-created" type="right" %}
+
+### Guidebook sections
```

Comment:
I love the cards here! re: the fonts on the cards - these are a bit of an eye-strain for me, and I think it's a combination of size, weight, and color. would it be possible to make the title text a smidge larger (and the dark purple from our palette), the sub-header smaller and//or italicized, and then make the links smaller? there are definitely other ways to approach it, but not having a visual hierarchy makes this a bit difficult to parse.
<img width="420" alt="Screenshot 2024-01-25 at 1 05 51 PM" src="https://github.com/pyOpenSci/pyopensci.github.io/assets/23085445/fb94e25a-cc17-4999-b6a9-52c5a6f58aec">

Reply:
ok they now look like this . because of the way the styles are setup in order to modify that awkward space i'd have to modify sitewide styles. 

let's hold off on that until i have more time to see how those elements will apply throughout the site. But creating visual hierarchy within a card is easy! there is still a bit of tension between the card titles and the section header BUT i think this is good enough for a start at making it easier to skim! 

<img width="980" alt="Screenshot 2024-01-25 at 3 34 18 PM" src="https://github.com/pyOpenSci/pyopensci.github.io/assets/7649194/07bcbc1f-21ea-410a-990d-f9c23b7d39b5">

Reply:
oooh, this is awesome! it's definitely more clear and easier on the eyes!